pkg/dashboards/types.go

```diff
@@ -212,19 +212,19 @@ type DashboardUpdateResult struct {
 // DashboardWidgetConfigurationInput - Typed configuration for known visualizations. At most one may be populated.
 type DashboardWidgetConfigurationInput struct {
```

Comment:
@zlesnr This is a good one to think about how to handle in tutone.  I.e. we want all the nullable structs within this struct to be pointers.

Reply:
I could've sworn we account for nullable fields with pointers somewhere, but something could be slightly different with introspection on this.